hapic/ext/aiohttp/context.py

```diff
@@ -105,10 +101,12 @@ def __init__(
                 # Parse each managed exceptions to manage it if must be
                 for handled_exception in self._handled_exceptions:
                     if isinstance(exc, handled_exception.exception_class):
-                        dumped_error = \
-                            self._get_dumped_error_from_exception_error(exc)
+                        dumped_error = self._get_dumped_error_from_exception_error(
```

Comment:
maybe rename dumped_error to error so that pep8 is ok on one line (and `dumped` prefix seems not being required here)

Reply:
not enought place with `error` var name :/

Reply:
Do i rename `err` ?

Reply:
yes you can :)